hyperspy/docstrings/plot.py

```diff
@@ -88,20 +88,65 @@
 
         Parameters
         ----------
-        %s""" % BASE_PLOT_DOCSTRING_PARAMETERS
+        """
+
 
 PLOT1D_DOCSTRING = \
-"""norm : str, optional
+    """norm : str, optional
             The function used to normalize the data prior to plotting.
             Allowable strings are: ``'auto'``, ``'linear'``, ``'log'``.
             (default value is ``'auto'``).
             If ``'auto'``, intensity is plotted on a linear scale except when
             ``power_spectrum=True`` (only for complex signals).
+        autoscale : str
+            Set the axes to be reset to their maxima when the navigation
+            indices or data are changed. If 'x' is in the string, the
+            horizontal axis is reset automatically. If 'z' is in the string,
+            the vertical axis is reset automatically. Default is 'z'.
```

Comment:
To align with its signal2D counterpart:

The string must contain any combination of the 'x' and 'z'
characters. If 'x' or 'z' are in the string, the corresponding
axis limits are set to cover the full range of the data at a given
position and the axis limits will reset when the data or the
navigation indices are changed. Default is 'z'.

Reply:
The docstring looks good, but the usage of 'z' seems counter-intuitive for me, couldn't that be 'y'? I know what's the rationale behind your choice, but it seems weird to name an axis 'z' when there is no y-axis...

Reply:
Indeed, and I like that autoscale='z' for plotting signal1D and signal2D does the same thing. Is there something different from `z` and `y` which could be used?

Reply:
Well, maybe "i" for intensity. I find "z" more straightforward, but maybe that's just me. I think that it would be good to have some more opinions on this if possible. @hyperspy/developers, any opinion on this?

Reply:
I was also thinking about 'i' but I wasn't really convinced.

Reply:
Yes, the issue with "i" is that a code that contains "yi" is not identified immediately by most people as some sort of axes configuration, while for "yz" it is easier to make a winning guess of what it means.

Reply:
What's about `v` for values? It fits well with `vmin` and `vmax`, which is used for images. Then it would:
- `xv` for signal2D
- `xyv` for signal2D

Reply:
I would prefer ``i`` in that case, however my favourite remains ``z``. Let's try to break the gridlock, @tjof2, what do you think?

Reply:
Ha, I generally see `z` as meaning "depth", so would be bad for any future 3D plotting, while `v` is good for consistency with `vmin` and `vmax`.

Reply:
Good point, I agree.

Reply:
For example say there was a theoretical `Signal3D` you could do `xyzv` as a string then as @ericpre says